extensions/common/vault/hashicorp/vault-hashicorp/src/main/java/org/eclipse/edc/vault/hashicorp/client/HashicorpVaultTokenRenewService.java

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2022 Mercedes-Benz Tech Innovation GmbH
+ *  Copyright (c) 2025 Cofinity-X
```

Comment:
copyright

Reply:
I think GitHub got confused which file was old and which was new after I changed the module structure (since all files where moved then) 😄 reverted that part now and it fits again